sklearn/ensemble/_voting.py

```diff
@@ -87,7 +92,7 @@ def fit(self, X, y, sample_weight=None):
                 clone(clf),
                 X,
                 y,
-                sample_weight=sample_weight,
+                fit_params=fit_params,
```

Comment:
then you also need to tune the code for `_fit_single_estimator` so that it extracts `sample_weight` from `fit_params` instead of accepting it separately.